src/Sequence.cc

```diff
@@ -291,17 +323,16 @@ void Sequence::execute(Context& context, CommChannel* comm)
 
     context.step_setup_script = step_setup_script_;
 
-    send_message(comm, Message::Type::sequence_started, "Sequence started",
-                 Clock::now(), 0);
+    send_message(comm, Message::Type::sequence_started, cat(exec_block_name, " started"),
+                 Clock::now(), gul14::nullopt);
```

Comment:
!!! That was a 'bug', right? (the zero)

Reply:
Kind of. Actually, the step index in this message is not used anywhere. But IMO it is more logical to use `nullopt` in this case.